protoc_plugin/lib/src/protobuf_field.dart

```diff
@@ -273,21 +283,35 @@ class ProtobufField {
         if (baseType.isMessage || baseType.isGroup) {
           named['subBuilder'] = '$type.create';
         } else if (baseType.isEnum) {
+          // TODO: remove this branch.
```

Comment:
I wonder if we should open an issue or at least assign a name to this TODO.
Or perhaps expand it - why can this be removed? (not this PR)